install_config/aggregate_logging.adoc

```diff
@@ -326,6 +335,7 @@ the privileged SCC, similarly as for Fluentd above.
 You can use any volume type, such as
 link:../install_config/persistent_storage/persistent_storage_nfs.html[NFS].
 
+[[node_selector]]
```

Comment:
Use hyphens for anchors. Also preferably more unique (they're shared across all books in the set):

`logging-node-selector`